### ISSUE TITLE
AVT 1 - Accordion has DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4405]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP Version 1.8.0.0 - IBM Accessibility WCAG 2.1 Sept. 2019 Ruleset

## Description
Go to the React Accordion component
Expand all sections
Start DAP and the following violations are present (see screenshot):
<img width="1107" alt="Screen Shot 2019-10-21 at 1 18 23 PM" src="https://user-images.githubusercontent.com/21676914/67231738-8aab3280-f405-11e9-9730-fd2ccd5f3011.png">

**Comment by dakahn** [comment]
Think there's a problem with our Storybook story here -- the third section with the Select
